src/App.js

```diff
@@ -1,25 +1,84 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { useState } from 'react';
 
-function App() {
+import contacts from './contacts.json'
+
+
+
+function App() { 
+  const fiveContacts = contacts.slice(0,5)
+  
+  const [addContact, setAddContact] = useState(fiveContacts)
+  
+  const randomContact = (c) =>{ 
```

Comment:
Aca podriamos agregar una validacion para o tener contactos repetidos 👀 \
```suggestion
const randomContact = (c) => {
    const getRandomContact = c[Math.floor(Math.random() * c.length)];
    if (!addContact.includes(getRandomContact)) {
      setAddContact([...addContact, getRandomContact]);
    }
  };```